### ISSUE TITLE
Can I use Websockets? 

### ISSUE THREAD

**Issue by frankdeno** [dimitrov-d/cloud-function-template#1]
### Hola :wave:

Quick question.
**Can I deploy a cloudfunction that uses Websockets?**

Elaboration:
There is Open AI Realtime API. I want to use a Relay Server to hide my API Key. 
That allows me to not expose my API Key to the Client. I copied nicely the given example and deploy it on deno deploy. 
Works nicely. 

Official Example: https://github.com/openai/openai-realtime-console/tree/main/relay-server
My Example: https://gitlab.com/frankbevr/zauber/-/tree/main/apps/relay-server

That should be deployable via the provided Cloud functions.
Currently I don't grasp fully of what is going up, so its a bit cumbersome to bring my head to the point to just make the right edits in the index.js

Basically I have to include two line lines
[`const relay = new RealtimeRelay(OPENAI_API_KEY);`
`relay.listen(PORT);`](https://gitlab.com/frankbevr/zauber/-/blob/main/apps/relay-server/index.js?ref_type=heads#L16)

But where to do what 
and then whats next 
and on the client which url do I use to connect 
and  ??? 

The whole thing is not essential. Before I drop the Idea, I drop a Issue. 
It could be just a two liner solution with the right head. 

#### Cheers :v: 
P.s. [The Official](https://github.com/Apillon/cloud-function-template) doesn't have a issue tab, so I drop it in here

**Comment by dimitrov-d** [comment]
Hi @frankdeno, at this stage through Apillon's cloud functions we only support serverless cloud functions which are meant to execute upon a given trigger, which is why we give our users an HTTP endpoint that they are supposed to call when they want to execute the function, with support for also passing arbitrary data to it.

For that reason running websockets might not be the ideal use case for running via the cloud functions.

I suggest you read more on [our cloud functions wiki](https://wiki.apillon.io/web3-services/8-web3-cloud-functions.html) so you may get a better idea of how the service works.

**Comment by frankdeno** [comment]
Gotcha! Thanks, lesson learned :+1: 

Regardings to the wiki. I created Podcast, its a bit easier to consume than reading.  
Sadly it didn't help to answer my questions, but to get a gist of it quite neat.
Its my latest way to consume the latest technologies, its a breeze to consume. 
[🎧 Listen to Audio Podcast about Apillion - Cloud Functions.](https://notebooklm.google.com/notebook/409ce298-c3d2-4a98-9381-16681d01932b/audio)
So you know. I thought drop it in here :tipping_hand_man: 

Thanks for your response :saluting_face: 

Cheers :v:
